Modix.Data.Test/Sandbox.cs

```diff
@@ -1,27 +1,11 @@
 ﻿using System;
```

Comment:
This whole file should just be removed, honestly. It was something I made as an easy entry point to fuck around with some EF queries, early on.